src/styles/stars.scss

```diff
@@ -0,0 +1,22 @@
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  background: url("../images/star.svg");
+  background-repeat: no-repeat;
+  background-position: center;
+
+  width: 16px;
+  height: 16px;
+  margin-bottom: 0;
```

Comment:
you have margin-bottom by default? If yes - remove it in `reset.scss`
Check all code
```suggestion
```